tycho-packaging-plugin/src/main/java/org/eclipse/tycho/packaging/PackageFeatureMojo.java

```diff
@@ -96,6 +97,28 @@ public class PackageFeatureMojo extends AbstractTychoPackagingMojo {
     @Parameter(defaultValue = "${project.basedir}/" + FEATURE_XML)
     private File featureFile;
 
+    /**
+     * Additional files to be included in the feature <code>.jar</code>.
+     * <p>
+     * This can be used when <code>bin.includes</code> in <code>build.properties</code>
+     * is not flexible enough , e.g., for generated files.
+     * <p>
+     * If conflicting, additional files win over <code>bin.includes</code>. Example:<br>
+     *
+     * <pre>
+     * &lt;additionalFileSets&gt;
+     *  &lt;fileSet&gt;
+     *   &lt;directory&gt;${project.build.directory}/mytool-gen/&lt;/directory&gt;
+     *   &lt;includes&gt;
+     *    &lt;include&gt;&#42;&#42;/*&lt;/include&gt;
+     *   &lt;/includes&gt;
+     *  &lt;/fileSet&gt;
+     * &lt;/additionalFileSets&gt;
+     * </pre>
+     */
+    @Parameter
+    private DefaultFileSet[] additionalFileSets;
```

Comment:
should this not beeing inherited from the Abstract mojo?

Reply:
@laeubi definitely a good idea since . However the `package-iu` mojo would not use it, while still having it in its configuration.

Reply:
(If what you meant was moving it to `AbstractTychoPackagingMojo`).

Reply:
I must confess I have no idea for what one needs `package-iu` ... but think it would not be a big deal having one useless paramter there.

Reply:
@laeubi agree, at one point we might want to support addition file sets in that ui mojo too maybe. I'll move it if that's ok.

Reply:
@laeubi done. I've re-touched to documentation. Let me know if it's ok.